.eslintrc.js

```diff
@@ -1,26 +1,22 @@
 module.exports = {
 	extends: [
-		'wordpress',
-		'plugin:wordpress/esnext',
-		'plugin:@elementor/editor/recommended',
 		'plugin:react/recommended',
 		'plugin:no-jquery/deprecated',
+		'plugin:@wordpress/eslint-plugin/recommended-with-formatting',
 	],
 	plugins: [
-		'wordpress',
 		'babel',
 		'react',
-		'@elementor/editor',
```

Comment:
Why? It's used to ensure translation-ready on JSX syntax.

Reply:
https://github.com/elementor/elementor/pull/18596/commits/74236ae5d9537474d4854e23709854cc09cf89af